sql/moz-fx-data-shared-prod/search_derived/search_clients_daily_v8/query.sql

```diff
@@ -293,10 +192,7 @@ windowed AS (
     scalar_parent_urlbar_searchmode_topsites_urlbar_sum,
     scalar_parent_urlbar_searchmode_touchbar_sum,
     scalar_parent_urlbar_searchmode_typed_sum,
-    SAFE_SUBTRACT(
-      UNIX_DATE(DATE(SAFE.TIMESTAMP(subsession_start_date))),
-      profile_creation_date
-    ) AS profile_age_in_days,
+    profile_age_in_days,
     SUM(IF(type = 'organic', count, 0)) OVER w1 AS organic,
     SUM(IF(type = 'tagged-sap', count, 0)) OVER w1 AS tagged_sap,
```

Comment:
Do these need to be sums? I assume that the search count aggregation in clients_daily ends up giving us an array where each engine/source combination will be unique. So maybe we don't need to do any windowing or aggregation here at all?

Reply:
Based on my reading of `udf.search_counts_map_sum`, we'll indeed be guaranteed that the `flattened` CTE here will not have more than one row per `client_id, engine, source`, so we shouldn't need this aggregation step any more.

Reply:
Ben points out that we do engine normalization, etc. in this query, so the values here are not exactly what's in the array from clients_daily, and we're not necessarily guaranteed that two different values haven't been normalized to the same thing. So we do need aggregation.